src/kv/test/kv_serialisation.cpp

```diff
@@ -236,112 +212,19 @@ TEST_CASE_TEMPLATE(
 }
 
 struct CustomClass
-{
```

Comment:
This file is mostly a find and replace, plus deletions like this for things that only tested the old implementation.